src/pages/groupComparison/Mutations.tsx

```diff
@@ -0,0 +1,206 @@
+} from './GroupComparisonUtils';
+import AsyncSelect from 'react-select/async';
+import {
+    numberOfLeadingDecimalZeros,
+    formatPercentValue,
+} from 'cbioportal-utils';
+import { countUniqueMutations } from 'shared/lib/MutationUtils';
+
+interface IMutationProps {
+    store: GroupComparisonStore;
+    mutations?: Mutation[];
+    filters?: any;
+}
+
+@observer
+export default class Mutations extends React.Component<IMutationProps, {}> {
+    private mutationMapperToolStore: MutationMapperToolStore;
+    private groupComparisonMutationMapper:
+        | GroupComparisonMutationMapper
+        | undefined;
+
+    constructor(props: IMutationProps) {
+        super(props);
+        this.mutationMapperToolStore = new MutationMapperToolStore(
+            this.props.mutations ||
+                _(this.props.store.mutationsByGroup.result!)
+                    .values()
+                    .flatten()
+                    .value(),
+            this.props.filters || {
+                groupFilters: _(this.props.store.mutationsByGroup.result!)
+                    .keys()
+                    .value()
+                    .map(group => ({
+                        group: group,
+                        filter: {
+                            type: 'GroupComparisonFilter',
+                            values: [group],
+                        },
+                    })),
+                filterAppliersOverride: {
+                    ['GroupComparisonFilter']: this.props.store
+                        .applySampleIdFilter,
+                },
+                countUniqueMutations: this.countUniqueMutations,
+            }
+        );
+    }
+
+    @computed
+    get showPercent() {
+        return this.groupComparisonMutationMapper
+            ? this.groupComparisonMutationMapper.showPercent
+            : true;
+    }
+
+    @autobind
+    protected countUniqueMutations(mutations: Mutation[]) {
+        return this.showPercent
+            ? (countUniqueMutations(mutations) /
+                  this.props.store.countProfiledSamples.result!) *
+                  100
+            : countUniqueMutations(mutations);
+    }
+
+    @autobind
+    protected plotLollipopTooltipCountInfo(
+        count: number,
+        mutations?: Mutation[]
+    ): JSX.Element {
+        const decimalZeros = numberOfLeadingDecimalZeros(count);
+        const fractionDigits = decimalZeros < 0 ? 1 : decimalZeros + 2;
+        return mutations && mutations.length > 0 && this.showPercent ? (
+            <strong>
+                {formatPercentValue(count, fractionDigits)}% mutation rate
+            </strong>
+        ) : (
+            <strong>
+                {count} mutation{`${count !== 1 ? 's' : ''}`}
+            </strong>
+        );
+    }
+
+    @autobind
+    private onMutationMapperInit(
+        mutationMapper: GroupComparisonMutationMapper
+    ) {
+        this.groupComparisonMutationMapper = mutationMapper;
+    }
+
+    public render() {
```

Comment:
we want to use MobxPromiseView to await promise specific to this tab and also provide the loader.